client_app/src/components/shelter/ShiftDetails.js

```diff
@@ -7,17 +7,33 @@ import { TimePicker } from '@mui/x-date-pickers/TimePicker';
 import dayjs from 'dayjs';
 import dayjsUTC from 'dayjs/plugin/utc';
 import dayjsTimezone from 'dayjs/plugin/timezone';
-import ShiftDetailsTable from "./ShiftDetailsTable";
+import { ShiftDetailsTable } from "./ShiftDetailsTable.js";
+import { MoreDetailsBox } from "./MoreDetailsBox.js";
+import { ShiftsModal } from "./ShiftsModal.js"
 
 dayjs.extend(dayjsUTC);
 dayjs.extend(dayjsTimezone);
 
-function ShiftDetails() {
+export const ShiftDetails = () => {
+    const [isMModalOpen, setIsMModalOpen] = React.useState(false);
```

Comment:
Consider using import { useState } from "react", so you need to type React every time you want to use useState.